hadoop-tools/hadoop-azure/src/test/resources/log4j.properties

```diff
@@ -26,6 +26,8 @@ log4j.logger.org.apache.hadoop.fs.azure.AzureFileSystemThreadPoolExecutor=DEBUG
 log4j.logger.org.apache.hadoop.fs.azure.BlockBlobAppendStream=DEBUG
 log4j.logger.org.apache.hadoop.fs.azurebfs.contracts.services.TracingService=TRACE
 log4j.logger.org.apache.hadoop.fs.azurebfs.services.AbfsClient=DEBUG
+log4j.logger.org.apache.hadoop.fs.azurebfs.services.ReadBufferManager=TRACE
```

Comment:
Was this added for testing as this might add a lot of logging ?

Reply:
it was...but we run the rest of the tests logging at debug, and i don't see the prefetcher being any chattier than the rest of the stack.